src/components/Alert/AlertNotification.tsx

```diff
@@ -3,23 +3,25 @@ import {Alert} from 'react-bootstrap';
 import styles from './AlertNotification.module.scss';
 
 
-type AlertNotificationTypes = 'info' | 'warning' | 'notice';
+export type AlertNotificationTypes = 'info' | 'warning' | 'notice';
 
 interface Props {
   type: AlertNotificationTypes;
+  description: string;
 }
 
-export function AlertNotification ({type}: Props) {
+export function AlertNotification (alertInfo: Props) {
 
     const [show, setShow] = useState(true);
```

Comment:
This should be removed and just return the component.

Reply:
If you take that our it causes an error with type. As for the setshow it has to be set to UseState true or it returns an error and will not show.